src/app/clusters/on-off-server/on-off-im.cpp

```diff
@@ -0,0 +1,34 @@
+
+#include <app/CommandHandler.h>
+#include <app/CommandSender.h>
+#include <app/MessageDef/MessageDef.h>
+#include <app/im-encoder.h>
+#include <app/util/basic-types.h>
+
+#include <support/logging/CHIPLogging.h>
+
+namespace chip {
+namespace app {
+namespace cluster {
+namespace OnOff {
+
+// TODO: Call handler in on-off.cpp here.
+void HandleOffCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj)
+{
+    EncodeOffCommand(apCommandObj, 1, 0);
```

Comment:
I do not understand why we need to call `EncodeOffCommand` here ? Similar questions for the other handlers.

Also constants would be helpful to read the code instead of raw `1` and `0`.